dandi/files/bases.py

```diff
@@ -453,6 +456,14 @@ def get_metadata(
         metadata.path = self.path
         return metadata
 
+    def _get_inspector_versions(self):
+        url = "https://pypi.org/pypi/nwbinspector/json"
+        data = json.loads(requests.get(url=url).text)
```

Comment:
consider using etelemetry from @satra et al: https://pypi.org/project/etelemetry/ which we already use for dandi-cli itself. Doesn't need to be in the same PR

Reply:
Tried,

```python
etelemetry.get_project(repo="NeurodataWithoutBorders/nwbinspector")
```

results in

```python
RuntimeError: Connection to server could not be made
```

Reply:
indeed still down, eh https://github.com/sensein/etelemetry-client/issues/35  attn @satra

Reply:
should be back up now.

Reply:
Using `etelemetry` now

It simplified enough lines so that there's really no point to having it be a separate function, so I moved all the logic into the `try` of the logger-wrapping section